tests/utils/test_file.py

```diff
@@ -78,6 +78,29 @@ def test_open_maybe_zipped_archive(self):
 
 
 class TestListPyFilesPath(unittest.TestCase):
+    def setUp(self):
```

Comment:
Lets migrate this to pytest https://docs.pytest.org/en/stable/how-to/tmp_path.html#tmp-path

Something like:

 ```suggestion
class TestListPyFilesPath():
    @pytest.fixture(autouse=True)
    def test_dir(self, tmpdir):
        ...
```

Reply:
Done